crates/sui-types/src/transaction.rs

```diff
@@ -113,6 +116,22 @@ pub enum ObjectArg {
     Receiving(ObjectRef),
 }
 
+#[derive(Debug, PartialEq, Eq, Hash, Clone, Serialize, Deserialize)]
+pub struct BalanceWithdrawArg {
```

Comment:
I think we should settle on a name for the module before naming this call arg (or just be prepared to rename it)

Reply:
Happy to rename to whatever